### PR TITLE
Fixing install of CLU files

### PR DESCRIPTION
**Pull request by roigcarlo**
Here it goes. 

As we spoke, this will be only temporal until I know how data files are handled in python modules and we implement the correct solution.

Probably we will make a data folder or something like this.

### CODE REVIEW

File: applications/DEMApplication/CMakeLists.txt [hunk 1]
```diff
@@ -139,6 +139,11 @@ endif(USE_COTIRE MATCHES ON)
 # Add to the KratosMultiphisics Python module
 kratos_python_install(${INSTALL_PYTHON_USING_LINKS} ${CMAKE_CURRENT_SOURCE_DIR}/DEMApplication.py KratosMultiphysics/DEMApplication/__init__.py )
 
+# DEM: CLU files need to be installed.
+# This is a temporal fix until we find how standard python modules handle data files.
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements DESTINATION applications/${CURRENT_DIR_NAME} FILES_MATCHING PATTERN "*.clu" PATTERN ".svn" EXCLUDE)
+endif(${INSTALL_PYTHON_FILES} MATCHES ON)
```
Comment (maceligueta) [inline applications/DEMApplication/CMakeLists.txt]: It works if I remove line 145 (endif does not correspond to any if).

**Review by maceligueta — CHANGES_REQUESTED** [review]

File: applications/DEMApplication/CMakeLists.txt (hunk 1 above)
Comment (roigcarlo) [inline applications/DEMApplication/CMakeLists.txt]:
```suggestion

```

Comment (roigcarlo) [reply applications/DEMApplication/CMakeLists.txt]: Sorry, copypaste

**Review by maceligueta — APPROVED** [review]

**Comment by roigcarlo** [comment]
This should be added to the Release 7.1

**Pull request merged**